tensorflow/c/eager/c_api.h

```diff
@@ -350,6 +368,8 @@ TF_CAPI_EXPORT extern void TFE_OpSetAttrFunctionList(TFE_Op* op,
                                                      const TFE_Op** value,
                                                      int num_values);
 
+TF_CAPI_EXPORT extern void TFE_OpInferSingleInputTypeAttrs(TFE_Op* op);
```

Comment:
Comment what this function does?

Reply:
Oops, left over from a previous implementation, shouldn't be declared in `c_api.h`